src/olympia/constants/base.py

```diff
@@ -355,6 +355,12 @@
 # Dicts are not compatible with Firefox for Android, only desktop is relevant.
 DEFAULT_WEBEXT_DICT_MIN_VERSION_FIREFOX = '61.0'
 
+# The default version of Firefox that (is planned to) support manifest version 3 (MV3)
+# This will probably have to be adjusted later.
+DEFAULT_WEBEXT_MIN_VERSION_MV3_FIREFOX = '100.0'
+# We don't know if the Android min version will be different, but assume it might be.
+DEFAULT_WEBEXT_MIN_VERSION_MV3_ANDROID = '100.0'
```

Comment:
So maybe (if that's possible):

```suggestion
# We don't know if the Android min version will be different, but assume it might be.
DEFAULT_WEBEXT_MIN_VERSION_MV3_ANDROID = DEFAULT_WEBEXT_MIN_VERSION_MV3_FIREFOX
```